src/connectors/Excel.ts

```diff
@@ -0,0 +1,89 @@
+/// <reference types="node" />
```

Comment:
Why the need to reference the node declaration? Also, maybe is a good time to include the license?

Reply:
This is because of the compiler, we have made any separation of sources and directives